src/modules/form/components/DynamicField.tsx

```diff
@@ -411,20 +428,28 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
     case ItemType.Image:
       return (
         <InputContainer {...commonContainerProps}>
-          <Uploader
-            id={linkId}
-            image
-            onUpload={async (upload) => onChangeValue(upload.blobId)}
-          />
+          <LabelWithContent label={item.text} helperText={item.helperText}>
```

Comment:
Note that Uploaders in Assessments were previously not labelled. With this change, the Client Files form will now have the new text "Attachment". It looks like this:
![Screenshot 2024-11-21 at 5 18 02 PM](https://github.com/user-attachments/assets/51af011e-8a37-4a31-a293-47d4d46a0a14)

Reply:
Is "Attachment" a default label or is it just the value of `item.text` in this example? (i.e. if `text` was left empty, would there be no label)?

Reply:
It is the value of `item.text` defined here: https://github.com/greenriver/hmis-warehouse/blob/681544cdafe35c9ec96ccc0e706a14cb371382a3/drivers/hmis/lib/form_data/default/records/file.json#L19

Yes, if `text` was left empty there would be no label.